Rakefile

```diff
@@ -46,8 +46,6 @@ task :default => :test
 
 RDoc::Task.new do |rdoc|
   rdoc.title = "IRB"
-  rdoc.rdoc_files.include("*.md", "lib/**/*.rb")
-  rdoc.rdoc_files.exclude("lib/irb/xmp.rb")
```

Comment:
If we use these options, `.document` will be ignored. Since `.document` will be applied to `ruby/ruby`'s documentation generation too, we should prioritize using it.

Reply:
Actually, this is not correct. I think it'd be better if that's the case though. Will see how I can make it happen.

Reply:
@tompng I opened https://github.com/ruby/ruby/pull/12325 for this. Can you review it there too?